DigitalLearningSolutions.Web.Tests/TestHelpers/AvailableCourseHelper.cs

```diff
@@ -16,7 +16,8 @@ public static class AvailableCourseHelper
             string brand = "Brand 1",
             string? category = "Category 1",
             string? topic = "Topic 1",
-            int delegateStatus = 0
+            int delegateStatus = 0,
+            int hideInLearnerPortal =0

```

Comment:
The VS auto formatter should have added a space after the = here. Please could you run autoformat on any files changed?